src/main/java/com/conveyal/r5/transit/fare/FareTable.java

```diff
@@ -47,7 +54,9 @@ public Fare lookup (String from, String to) {
 
     public Fare lookup (Stop from, Stop to) {
         //TODO: how does this works WRT different GTFS feeds having Stops with same Ids?
-        return lookup(from.stopId, to.stopId);
+        return this.ignoreAgencyId
+                ? lookup(from.stopId.split(":")[1], to.stopId.split(":")[1])
```

Comment:
I'm wondering if we should use a regex here so we can catch everything after the first colon, in case a stop ID contains a colon (so you'd have a feed-qualified stop id of, say, `WMATA:GalleryPlace:LowerLevel`), where the stopId is `GalleryPlace:LowerLevel`.